vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp

```diff
@@ -42,6 +42,28 @@ function drawLinks() {
         $("#space").append(link);
     }
     </c:forEach>
+    
+    <c:forEach items="${externalLinkList}" var="link">
+    {
+        var posX = parseInt($("#space").css('margin-left')) + $("#space").position().left; 
+        var posY = $("#space").position().top;
+        var link = $('<a></a>');
+        link.attr('href', '<c:url value="http://${link.externalLink.externalLink}"/>');
```

Comment:
let's open external links in a new tab/window

Reply:
I doubt that this will work. why did you do that?

Reply:
for the below code, i tried with external link - "http://www.google.com" and it opened in new tab. What is you suggestion to do it in another way?

link.attr('href', '<c:url value="${link.externalLink.externalLink}"/>');
link.attr('target', "_blank");

Reply:
where is the target=_blank set?

Reply:
also, external links will be complete urls, no c:url necessary.